go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/prebid/prebid-server
 
-go 1.16
+go 1.19
```

Comment:
Please submit this change as a separate PR to master. There are other files which need to be updated too, such as the github test workflows and the docker files.